Lib/fontTools/varLib/instancer.py

```diff
@@ -1284,6 +1298,158 @@ def instantiateVariableFont(
     return varfont
 
 
+def axisValuesFromAxisLimits(stat, axisLimits):
+
+    axisRecords = stat.table.DesignAxisRecord.Axis
+    axisValues = stat.table.AxisValueArray.AxisValue
+
+    format4 = [a for a in axisValues if a.Format == 4]
+    nonformat4  = [a for a in axisValues if a not in format4]
+    axisValues = format4 + nonformat4
+
+    axisOrder = {a.AxisOrdering: a.AxisTag for a in axisRecords}
+    pinnedAxes = set(k for k, v in axisLimits.items() if isinstance(v, (float, int)))
+
+    results, seen_axes = [], set()
+    for axisValue in axisValues:
+
+        # Ignore axisValue if it has ELIDABLE_AXIS_VALUE_NAME flag enabled.
+        # Enabling this flag will hide the axisValue in application font menus.
+        # TODO this is too greedy! we need to retain wght axisValues
+        if axisValue.Flags == 2:
```

Comment:
Can you define constants for the flags and then use that?
`if axisValue.Flags == ELIDABLE_AXIS_VALUE_NAME:`

Reply:
Will do thank you!